py_gtfs_rt_ingestion/batch_files.py

```diff
@@ -8,6 +8,7 @@
 from collections.abc import Iterable
 from typing import NamedTuple
 
+from py_gtfs_rt_ingestion import DEFAULT_S3_PREFIX
```

Comment:
i added this as a hard coded value in the library. the more i messed around in here, the more i felt like we should just handle this in env vars, but i didn't want to open another PR in the devops repo.